pkgs/applications/networking/mailreaders/notmuch/default.nix

```diff
@@ -18,6 +18,8 @@ stdenv.mkDerivation rec {
     sha256 = "wfLO7kf2iXESItcgWvKj/npKnYwy5OCyStZviN9qR9M=";
   };
 
+  patches = [ ./test-fix-support-for-gpgsm-in-gnupg-2.3.patch ];
```

Comment:
Can you add a link to the upstream PR/ML, so we can track when it has landed upstream?

Reply:
notmuch accepts contributions over email, this is the `git format-patch` that was emailed to them.

Reply:
added link to mailing list post